src/rpc/rawtransaction.cpp

```diff
@@ -1771,6 +1810,103 @@ UniValue rawissueasset(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue rawreissueasset(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 2)
+        throw runtime_error(
+            "rawreissueasset transaction {\"vin\":\"n\", \"asset_amount\":x.xxx, \"asset_address\":\"address\", \"asset_blinder\":<hex>, \"entropy\":<hex>, ( \"contract_hash\":<hex> )}\n"
+            "\nRe-issue an asset by attaching pseudo-inputs to transaction inputs, revealing the underlying reissuance token of the input. Returns the transaction hex.\n"
+            "\nArguments:\n"
+            "1. \"transaction\"           (string, required) Transaction in hex in which to include a peg-in input.\n"
```

Comment:
You mean issuance here instead of pegin?

Reply:
oops yes